scripts/mutect2_wdl/mutect2_multi_sample_template.json

```diff
@@ -14,9 +14,8 @@
   "Mutect2_Multi.variants_for_contamination_index": "$__variants_for_contamination_index__",
   "Mutect2_Multi.is_run_orientation_bias_filter": true,
   "Mutect2_Multi.is_run_oncotator": "$__is_run_oncotator__",
-  "Mutect2_Multi.m2_docker": "broadinstitute/gatk-protected:1.0.0.0-alpha1.2.4",
+  "Mutect2_Multi.gatk_docker": "broadinstitute/gatk-protected:1.0.0.0-alpha1.2.4",
```

Comment:
Update example docker image, please.  This is way out of date.

Reply:
done